src/parser/jobs/drg/modules/notes.md

```diff
@@ -26,4 +26,4 @@
 - [x] Chaos Thrust uptime and clipping (Debuffs.js)
 
 ### Rotation
-- [ ] Adhering to the standard 11-GCD rotation
+- [-] Adhering to the standard 11-GCD rotation (This has been removed.)
```

Comment:
You can cross this out with a strikethrough in another PR, not gonna block merging for it but just FYI. No idea what's up with the `/` and `-`, GitHub markdown just supports `-` as far as I know.